recipes/pysbol3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pysbol3" %}
+{% set version = "v1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SynBioDex/pySBOL3/archive/{{ version }}.tar.gz
+  sha256: 2f2c8305497a09677b4c1fb23219d8e1529e1be866ecf3d3973fff23c884630c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - rdflib >=6.1.1
+    - python-dateutil ~=2.8.2
+    - pyshacl ~=0.18.1
+
+test:
+  source_files:
+    - test
+    - examples
+  imports:
+    - sbol3
+  requires:
+    - pytest
+    - git
+  commands:
+    - git clone --recurse-submodules https://github.com/SynBioDex/pySBOL3.git && cp -r pySBOL3/test/SBOLTestSuite test/.
+    - pytest -v test
+
+about:
+  home: https://github.com/SynBioDex/pySBOL3
+  summary: "Pure Python implementation of SBOL 3 standard"
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - kenza12
+    - breakthewall
+    - tduigou
```

Comment:
We need @breakthewall and @tduigou OK here to merge this.

Reply:
Hi @ocefpaf, I'm OK to proceed

Reply:
Hi, ok to merge! Thanks!